m2cgen/assemblers/tree.py

```diff
@@ -3,8 +3,11 @@
 from m2cgen import ast
 from m2cgen.assemblers import utils
 from m2cgen.assemblers.base import ModelAssembler
-from sklearn import tree
-from sklearn.tree._tree import TREE_LEAF
+
+
+# refer to 
+# https://github.com/scikit-learn/scikit-learn/blob/fd12d5684ad224ad7760374b1dcca2821c644feb/sklearn/tree/_tree.pyx#L64
+TREE_LEAF = -1
```

Comment:
I'm absolutely sure that it doesn't worth to install huge library with own dependencies for one constant. 😃

Reply:
Yes, I 100% agree with you.